### DISCUSSION TITLE
7 Months still no access to Copilot

### DISCUSSION THREAD

**Discussion by avelev99** [community/community#11255] [Copilot Conversations] [answered]
I know that only a few developers are supposed to get access to the network in its testing stage, but I often see people get access 1-2 weeks after enrollment, I however enrolled about 7 months ago and yet I am still waiting to be whitelisted. Are there any requirements that would increase one's chance of being whitelisted?

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
